test/topology_custom/test_deprecating_cluster_features.py

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
Please add 'test:' to the subject.

Reply:
It's too late for this particular commit - it was a part of https://github.com/scylladb/scylladb/pull/14701 which got merged - but there are some other commits that touch test code and need a similar adjustment. Will fix them.